text-summarization/custom-recipes/text-summarization-compute/recipe.py

```diff
@@ -69,8 +75,11 @@ def summarize(text):
             text = text.lower()
             all_capital = True
         
-        parser = PlaintextParser.from_string(text.decode(
-            'ascii', errors='ignore'), Tokenizer(LANGUAGE))
+        if (sys.version_info > (3,0)):
+            parser = PlaintextParser.from_string(text, Tokenizer(LANGUAGE))
+        else:
+            parser = PlaintextParser.from_string(text.decode('ascii', errors='ignore'), Tokenizer(LANGUAGE))
```

Comment:
I am not familiar with this operation, but in the later version would there be any ill consequence of a call or is it just unnecessary? Even if it doesn't work, maybe handle this as a try-catch instead (nitpick)